app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@shubham-padia 
After looking at [PR#3763](https://github.com/fossasia/open-event-orga-server/pull/3763) , I sent both ( just for checking ).
Problem is that it's even saving timezone aware timestamps in UTC without the offset in the database.